cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -703,7 +705,7 @@ public void delete(Reconciliation reconciliation, Handler h) {
         };
 
         // Now try to reconcile all the Kafka clusters
-        ops.reconcileAll("test", clusterCmNamespace, Labels.EMPTY);
+        ops.reconcileAll("test", clusterCmNamespace, Labels.EMPTY).await();
```

Comment:
Maybe we want some timeout to prevent this test stuck.

Reply:
I think the tests have some implicit timeouts. That might be enough.